docs/docs/index.md

```
@@ -1,10 +1,107 @@
+  <a class="ak-btn ak-btn-secondary" href="https://github.com/AltimateAI/altimate-code" target="_blank" rel="noopener">View on GitHub</a>
+</div>
+
+<div class="nt-cards nt-grid cols-3" markdown>
+
+<div class="nt-card" markdown>
+<div class="nt-card-content" markdown>
+<div class="ak-card-icon">
+<svg viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"><path d="M13 2 3 14h7l-1 8 10-12h-7l1-8z"/></svg>
+</div>
+
+### [Quickstart](/getting-started/)
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)